main.go

```diff
@@ -77,17 +77,39 @@ func main() {
 			queryString = fmt.Sprintf("?%s", strings.TrimLeft(req.QueryString, "?"))
 		}
 
-		functionURL := fmt.Sprintf("http://%s%s:8080%s%s", req.Function, config.FunctionSuffix, path, queryString)
+		functionURL := fmt.Sprintf("http://%s%s:8080%s%s",
+			req.Function,
+			config.FunctionSuffix,
+			path,
+			queryString)
+
+		if config.GatewayInvoke {
+			functionURL = fmt.Sprintf("http://%s:8080/function/%s/%s%s",
```

Comment:
@alexellis with simply removing both back slashes here(note that it will be consistent with the function url in line 80) we'll fix the issue and it'll work correctly.

Reply:
Thank you! I'll add unit tests